docs/platforms.md

```diff
@@ -160,15 +160,23 @@ By default, `ARM64` is not enabled when running on non-ARM64 runners. Use [`CIBW
 
 Pyodide is offered as an experimental feature in cibuildwheel.
 
-### Prerequisites
+### System requirements
 
-You need to have a matching host version of Python (unlike all other cibuildwheel platforms). Linux host highly recommended; macOS hosts may work (e.g. invoking `pytest` directly in [`CIBW_TEST_COMMAND`](options.md#test-command) is [currently failing](https://github.com/pyodide/pyodide/issues/4802)) and Windows hosts will not work.
+Pyodide builds require a Linux or macOS machine.
 
 ### Specifying a pyodide build
 
 You must target pyodide with `--platform pyodide` (or use `--only` on the identifier).
 
-## iOS
+### Choosing a Pyodide version {: #pyodide-choosing-a-version}
+
+It is also possible to target a specific Pyodide version by setting the `CIBW_PYODIDE_VERSION` option to the desired version. Users are responsible for setting an appropriate Pyodide version according to the `pyodide-build` version. A list is available in Pyodide's [cross-build environments metadata file](https://github.com/pyodide/pyodide/blob/main/pyodide-cross-build-environments.json), which can be viewed more easily by installing `pyodide-build` from PyPI and use `pyodide xbuildenv search --all` to see a compatibility table.
+
+### Running tests
+
+Currently, it's recommended to run tests using a `python -m` entrypoint, rather than a command line entrypoint, or a shell script. This is because custom entrypoints have some issues in the Pyodide virtual environment. For example, `pytest` may not work as a command line entrypoint, but will work as a `python -m pytest` entrypoint.
```

Comment:
I think quite a few users will face breakages due to this, such as scikit-image, scikit-learn, and NumPy – but I think that is fine :)

Reply:
It would be nice if we could just fix these problems in pyodide-build. They probably aren't so hard to fix.

Reply:
That would be great. I think the pip entrypoint already works, so perhaps the approach used there could be replicated to package entrypoints.

Reply:
> Edit: never mind, I saw that there's `util.invoke_pytest()` that should help with it.

That's a util internal to our tests, not exposed to users.